fe/src/static/sass/base.scss

```diff
@@ -1,7 +1,53 @@
-$font-stack:    Helvetica, sans-serif;
+/***********************************************
+			DEFAULT STYLES
+***********************************************/
+
+@import url('https://fonts.googleapis.com/css?family=Montserrat');
+$font-stack: 'Montserrat', Helvetica, sans-serif;
+
+$dark-black: rgb(15, 15, 15);
+$light-black: rgb(22, 22, 22);
+$green: rgb(24, 177, 62);
 $primary-color: #333;
```

Comment:
? Don't include colors outside the Spotify assets until we have a legitimate need.